assets/airlines/tap.json

```diff
@@ -31,7 +31,7 @@
             ["A321", 4],
             ["E190", 13]
         ],
-        "airbusShort": [
+        "airbusshort": [
```

Comment:
use camelCase

Reply:
Completely removed the whole airbusshort-thing for TAP in the new build, and went with a tap/loww prefix, to get a A320+A321+A332 setup